app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
+
+<% if @resource.try(:unconfirmed_email?) %>
+  <p><%= t 'email_is_being_changed' @resource.unconfirmed_email %>.</p>
+<% else %>
+  <p><%= t 'email_has_been_changed' @resource.email %>.</p>
```

Comment:
you can move the @resource.try(:unconfirmed_email?) ? @resource.unconfirmed_email : @resource.email to a helper method.